spec/acceptance/basic_spec.rb

```diff
@@ -33,13 +33,19 @@
       class { 'corosync':
         multicast_address => '224.0.0.1',
         authkey           => '/tmp/ca.pem',
-        bind_address => $ipaddress
+        bind_address      => '127.0.0.1',
+        set_votequorum    => true,
+        quorum_members    => ['127.0.0.1'],
       }
       corosync::service { 'pacemaker':
         version => '1',
-      } ->
-      service { 'pacemaker':
-        ensure => running,
+      }
+      unless ($::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease, '14.04') >= 0) {
```

Comment:
that's an actual service declaration…
what we can put in params, is a variable that resolves to boolean true/false depending on whether pacemaker should be running.